ui/app.tsx

```diff
@@ -43,6 +43,17 @@ window.addEventListener('load', function () {
     return;
   }
 
+  // coldbrew: https://github.com/plasticityai/coldbrew
+  const coldbrew = document.createElement('script');
+  coldbrew.src = 'https://cdn.jsdelivr.net/gh/plasticityai/coldbrew@0.0.74/dist/coldbrew.js';
+  document.body.appendChild(coldbrew);
+
+  coldbrew.onload = async function () {
+    (window as any).coldbrew = await (window as any).Coldbrew.load();
+  };
```

Comment:
Once you bring this PR up to date you should move this into the shared/languages definition area into the function inMemoryInit. If you you look at the existing python definition on recent `main` you'll see the pyodide code there too.